### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by Sandy4321** [aksnzhy/xlearn#281]
Can it be used on Windows

**Comment by etveritas** [comment]
yep

**Comment by Sandy4321** [comment]
super , so how install this on windows?

**Comment by aksnzhy** [comment]
> super , so how install this on windows?

Hi, please see this link: https://xlearn-doc.readthedocs.io/en/latest/install/install_windows.html

Thanks!

**Comment by Sandy4321** [comment]
still not clear
do I need visual studio to use with 
Install xLearn from pip

in any case can you share
First, you need to build the executable files (xlearn_train.exe and xlearn_predict.exe), as well as the shared library (xlearn_api.dll for Windows) from the C++ code. After that, users need to install the xLearn Python Package

Users should make sure that choose the c++ development tools when install VS2017.
but will be vs2019 ok


can you give step by step instructions
it is not clear what instructions are nested and what not 

can you share something easy to install by using only pip
why others provide only pip installations and you can not ? 

to sum up
can be this link
https://github.com/aksnzhy/xlearn/releases
used to get windows .whl?
as written
Users can download this binary python package from tab release, then use pip command install the .whl file which you download.

can you share exact file and command to use
is it
pip xlearn-0.4.4-py2.py3-none-win_amd64.whl
??
where this binary ?
xLearn 0.4.3
@aksnzhy aksnzhy released this on Mar 25, 2019 · 165 commits to master since this release

Main Update:
*Fix bugs in previous version
*Provide binary python package on windows. Support Python(x64) in these versions: py2.7,py3.4,py3.5,py3.6,py3.7
